docs/enums/tableau.analyticsobjecttype.html

```diff
@@ -295,7 +296,7 @@ <h2>Legend</h2>
                 m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
         })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
 
-        ga('create', 'UA-625217-51', '');
+        ga('create', 'UA-625217-51', 'auto');
```

Comment:
Do we know why this line in the google analytics code changed?

Reply:
Interesting.  It doesn't look like we changed anything in the typedoc commnad in  package.json.  The  `auto` might refer to the GA site. WE don't use it, but the `-gaSite` option defaults to `auto`.

Reply:
Typedoc version was updated, I believe. These must just be new defaults.

Reply:
A Typedoc update might explain the relative paths being added too.